src/test/java/tests/FilterEvalTests.java

```diff
@@ -824,6 +825,40 @@ public void filterLabelMatching() {
         assertFalse(Qualifier.labelMatches(".", "p.high"));
     }
 
+    @Test
+    public void QualifierProcess_useInvalidUsername_getUserWarning() {
```

Comment:
It used to call `Qualifier.process` directly. Will change that.

Btw, In a case where a test method calls `Class.method()`, what is the correct test method name? Is it `classMethod_...`?